### DISCUSSION TITLE
APIM New Developer Portal Does not work Azure B2C SSO

### DISCUSSION THREAD

**Discussion by sakthimpwork** [Azure/api-management-developer-portal#2600] [Feature request]
**Every bug report should have precise description and reproduction steps; console traces or source code references are appreciated.**

**For assistance requests, contact Azure support or submit a post [on Stack Overflow](http://aka.ms/apimso). We don't provide support through GitHub Issues. Feature requests can be raised [on the Azure Feedback Forum](https://aka.ms/apimwish).**

## Bug description

Single Sign On does not work as expected for Azure B2C identity

## Reproduction steps

Pre-request: 
1. Create .Net Core 6 Web Application using Azure B2C identity authentication
2. Create New APIM Instance with Azure B2C authentication

1. Go to 'Web Application login with Azure B2C identity.'
3. Click on 'APIM Developer Portal Link' Which is taking me from my application to APIM new Developer portal
4. Still showing Sign In OAuth widget(AKS "Active Director B2C". But when I click the button manually, it got authenticated without any issue.

## Expected behavior

APIM New developer portal should sign in automatically since the user is already authenticated from a different application. Both are using the same identity provider, which is Azure B2C.

## Is your portal managed or self-hosted?

Managed

## Release tag or commit SHA (if using self-hosted version)

[e.g., release `2.0.0`, commit `c45da9778b70d369aba60fa2e63c191efe2b548f`]

##mypocapim

[e.g., contoso-api]

## Environment

 - Operating system:Windows [e.g., iOS]
 - Browser: Same behavior Chrome, Edge, Firebox [e.g., Google Chrome, Safari]
 - Version: [e.g., 22]

**Comment by ghost** [comment]
@sakthimpwork, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
@sakthimpwork, technically, session of AAD B2C and developer portal sessions are different, because we exchange B2C id_token to APIM-specific one. However, it's not impossible to implement, so I'll mark this issue as feature request.

**Comment by ghost** [comment]
@sakthimpwork, thank you for requesting this feature.
